### ISSUE TITLE
Invite Only Website has no spam protection and no email verfication.

### ISSUE THREAD

**Issue by PhpFoxJohnJr** [PHPfox-Official/phpfox-v4-issues#2323]
I set up my site as invite only.
I also set it up to verify the email address.
I logged in as a member I made with the membership of a registered user.
I then use the invite in the footer and invited a made-up email address.
123456@123456.com
and it said it was sent...ok

Then I opened another browser and went to the home page and put that email address in the invite box and it came up and allowed me to enter a Name, the same email address and a password.  I did that and bang...my account was made with no verification of email.  So I think a spammer could use excel spreadsheet and make a list of fake emails like obama@whitehouse.gov, trump@whitehouse.gov to something simple like 1@1.com, 2@1.com, 3@1.com up to 50 at a time and the system would let it pass through without verification that you even own that email address or worst yet...doesn't even exist.

What I expect to happen:  at least have email verification on a invite-only website.
phpfox version 4.6.1 b3
linux server

**Comment by spirogg** [comment]
@PhpFoxJohnJr ,

I tried this: 
- I made my site invite only, logged in and sent an invite to another email i own.
- then went to main site and added email i sent for invite
- then i added username email and password clicked submit.

- then i went back to login with my username and password and it would not allow me
 said i needed to verify my email first in order to login


Example:
admincp settings

**Verify Email At Signup: ON**
When a guest signs up, should they verify their email address? 
If enabled an email will be sent with a special link to verify that is their address.



**Invite Only: ON**
Enable this option if your community is an "invite only" community.

v4.6.1.b4 just fyi

**Comment by PhpFoxJohnJr** [comment]
Interesting...it did work.  Have to leave now but I am going to keep this issue open until I double check as I tested it using the invite link on the bottom to send the invite and then went to the home page to enter the fake email address to sign up and in (I am not using the email link as I am testing for fake emails) and it did come up and say email verification.  I have to leave but I will test it later as I think I made a private group and sent the invite out through the private group and that was the issue...but thanks...this give me hope to go back and retest this issues as I stopped all progress on my site for the last two months because of this.  Thanks again.

Here is the forum thread...if it does work I wished phpfox told me two months ago before I opened this ticket.  This is why we post on the forum first (talking to phpfox) so we don't open issues on the public github making the software look bad when it may be our individual site error.

https://community.phpfox.com/forum/thread/57255/big-loophole-for-spammer-to-flood-your-site-with-fake-members/
